server/src/main/java/io/crate/execution/ddl/tables/TransportOpenCloseTableOrPartitionAction.java

```diff
@@ -79,11 +84,10 @@ public ClusterStateTaskExecutor<OpenCloseTableOrPartitionRequest> clusterStateTa
 
     @Override
     protected ClusterBlockException checkBlock(OpenCloseTableOrPartitionRequest request, ClusterState state) {
-        return TransportCloseTable.checkBlock(
-            state,
-            request.ignoreUnavailableIndices(),
-            request.tables(),
-            request.partitionIndexName()
-        );
+        if (isEmptyPartitionedTable(request.tableIdent(), state)) {
```

Comment:
this reverted to https://github.com/crate/crate/pull/12231 - first change before https://github.com/crate/crate/commit/6857a28b669b6e6f056e59cbfdc760c59739701c and https://github.com/crate/crate/commit/562abc8a358c344e3986cc1f1a27e06cec65f43c + kept changes from https://github.com/crate/crate/pull/12370 and https://github.com/crate/crate/pull/12378